packages/flutter/lib/src/painting/_network_image_io.dart

```diff
@@ -122,3 +122,25 @@ class NetworkImage extends image_provider.ImageProvider<image_provider.NetworkIm
   @override
   String toString() => '$runtimeType("$url", scale: $scale)';
 }
+
+/// Thrown to indicate that loading a network image failed
+///
```

Comment:
Nits: remove blank line, and make the first line a complete sentence ending with a `.`.  Perhaps something like `The exception thrown when the HTTP request to load a network image fails.`